### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**
Initial PR

### CODE REVIEW

File: mayhem/Mayhemfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+project: pdfplumber
```
Comment (xansec) [inline mayhem/Mayhemfile:1]: Is the timeout required to be 8?

File: mayhem/fuzz_plumber.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+#! /usr/bin/env python3
```
Comment (xansec) [inline mayhem/fuzz_plumber.py:1]: Exception handling is non-deterministic.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: mayhem/Mayhemfile (hunk 1 above)
Comment (capuanob) [reply mayhem/Mayhemfile:1]: The defect it found took ~4.5 seconds, so I'll drop it to 6 to be safe. The PDF parsing is fairly expensive at its worst case

File: mayhem/fuzz_plumber.py (hunk 2 above)
Comment (capuanob) [reply mayhem/fuzz_plumber.py:1]: Made deterministic using an iteration counter that defers exception handling

**Comment by capuanob** [comment]
@xansec Resolved!

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**